.github/workflows/ci.yaml

```diff
@@ -101,19 +101,24 @@ jobs:
       #       . -> target
       #       ./crates/proc-macro-srv/proc-macro-test/imp -> target
 
-      - uses: taiki-e/install-action@nextest
+      - name: Install nextest
+        uses: taiki-e/install-action@nextest
 
       - name: Codegen checks (rust-analyzer)
         if: matrix.os == 'ubuntu-latest'
         run: cargo codegen --check
 
-      - name: Compile (tests)
+      - name: Compile tests
         run: cargo test --no-run
 
-      - name: Test
+      - name: Run tests
         run: cargo nextest run --no-fail-fast --hide-progress-bar --status-level fail
 
-      - name: clippy
+      - name: Cancel parallel jobs
+        if: failure()
+        uses: action-pack/cancel@v1
```

Comment:
I would be vary of using a third-party action for this, especially one that is not under the `actions` organization and doesn't seem to have a lot of history and stars. At the very least I would pin the used commit hash.

Reply:
we should be able to use the rest API endpoint github provides directly instead

Reply:
Done, thanks for looking into the API!